lib/common.py

```diff
@@ -97,8 +97,7 @@ def timeout(proc):
   logger("CALL: {} (pid: {})".format(command, proc.pid))
 
   reader = threading.Thread(target = readproc, args = [proc])
-  timer = threading.Timer(get_media().call_timeout, timeout, [proc])
-
+  timer = threading.Timer(get_media()._get_call_timeout(), timeout, [proc])
```

Comment:
Sorry, I just noticed this... there is another get_media().call_timeout usage on line 121 that needs updated too.

Reply:
Maybe on line 121, we should use timer.interval instead of get_media().call_timeout.

Reply:
under line 121, changed get_media().call_timeout to timer.interval

Reply:
I don't see the change.  Did you forget to push?

Reply:
sorry, my mistake can you check now